### PR TITLE
Added tests to the utility functions of the Tags class

### PR DESCRIPTION
**Pull request by Vaibhavi1707**
## Purpose 
Adds tests for utility functions in the Tags class.

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code

### CODE REVIEW

**Review by david-allison-1 — APPROVED** [review]
Looks good to me

File: AnkiDroid/src/test/java/com/ichi2/libanki/TagsTest.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
+package com.ichi2.libanki;
```
Comment (mikehardy) [inline AnkiDroid/src/test/java/com/ichi2/libanki/TagsTest.java:1]: Needs a copyright header - can you add one please?

**Review by mikehardy — CHANGES_REQUESTED** [review]

**Comment by mikehardy** [comment]
maintainer note: can be merged after addition of copyright header

File: AnkiDroid/src/test/java/com/ichi2/libanki/TagsTest.java (hunk 1 above)
Comment (Vaibhavi1707) [reply AnkiDroid/src/test/java/com/ichi2/libanki/TagsTest.java:1]: Sure

**Review by mikehardy — APPROVED** [review]
LGTM thank you! Note for @TarekkMA I just reviewed a massive TagsDialog refactor PR from you, I'm not sure if this will collide or not, but I thought I'd mention you in case it collided

**Comment by TarekkMA** [comment]
@mikehardy thank you for mentioning me, My PR did touch only the UI part. so I guess we are fine.

**Comment by mikehardy** [comment]
Hi there!

Just a friendly notice that we try to process OpenCollective payments monthly - it's time for April 2021 submissions

If you are interested in compensation for this work, the process with details is here:

https://github.com/ankidroid/Anki-Android/wiki/OpenCollective-Payment-Process#how-to-get-paid

(I only post one comment per person to avoid spamming you, regardless of the number of PRs merged, but this note applies to all PRs merged in April)

Thanks!

**Pull request merged**